plugins/kotlin/j2k/shared/tests/testData/newJ2k/methodCallExpression/stringConcatFormatting.kt

```diff
@@ -6,9 +6,9 @@ internal object A {
         val result = s
             .replace("_", "/") + "=="
 
-        val result2 = (s
-            .replace("_", "/") // comment
-                + "==")
+        val result2 = s
+            .replace("_", "/") + "==" // comment
+
```

Comment:
Would be nice to drop the extra line break

Reply:
I definitely agree, but unfortunately it's pretty hard to fix because of the way the printing phase is set up. In this case, the problem is that the `JKLocalVariable` node has 2 trailing newlines, and the comment node has 1 trailing newline.